opentech/apply/funds/views.py

```diff
@@ -345,6 +347,35 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+@method_decorator(staff_required, name='dispatch')
+class CreateProjectView(DelegatedViewMixin, CreateView):
+    context_name = 'project_form'
+    form_class = CreateProjectForm
+    model = Project
+
+    def form_valid(self, form):
+        submission_id = form.cleaned_data['submission']
+        try:
+            submission = ApplicationSubmission.objects.get(pk=submission_id)
+        except ApplicationSubmission.DoesNotExist:
+            return self.form_invalid(form)
+
+        project, created = Project.create_from_submission(submission)
```

Comment:
Should this be moved onto the form save method? then we do not need to customise this view as much.

Reply:
Fixed in 75b0a597